src/app/query/stacks/mempool/mempool.hooks.ts

```diff
@@ -75,3 +76,16 @@ export function useCurrentAccountMempoolTransactionsBalance() {
   );
   return createMoney(tokenTransferTxsBalance.plus(pendingTxsFeesBalance), 'STX');
 }
+
+export const SAME_NONCE_FEE_TO_PLUS = 0.000001;
```

Comment:
add*

Though, for readability, maybe we could just make a function that does this, and export that?

`export function increaseValueByOneMicroStx () {}` ?

so the other file reads value.fee = increaseValueByOneMicroStx(value.fee)